backend/models/song.js

```diff
@@ -1,12 +1,13 @@
 const mongoose = require('mongoose');
+const User = require('./user')
 
 const songSchema = mongoose.Schema({
     name: {type: String, require:true},
     genre: {type:String, require:true},
     song_path: {type: String, require:true},
     image_path: {type: String, require:true},
     release_date: {type: Date, require:true},
-    artists: [String], //TODO: change to artist array
+    artists: {type: [User.schema], require:true},
```

Comment:
The User.schema is a one to many relationship?

Reply:
in mongo it is not a real relationship, but in sql world yes